projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/_core-overrides.scss

```diff
@@ -77,3 +77,5 @@
 		color: $text-color;
 	}
 }
+
+@import "../_upsell-banner";
```

Comment:
Is this needed? Why is not enough to load it only in the `_overrides.scss` file?

Reply:
This was totally wrong, removed! efd046f